blacklist.yml

```diff
@@ -505,3 +505,5 @@ wetback: other
 wop: other
 "wrapping the weasel": other
 yclit: other
+
+vim: escescesc
```

Comment:
There is also `escescescesc` and `escescescescesc` but I think that this is a good start.